dags/openshift_nightlies/docs/manifest_and_releases.md

```diff
@@ -50,12 +49,19 @@ platforms:
         config:
           install: openstack/sdn.json
           benchmarks: openstack.json
-
-
 ```
 
 would create a total of 8 dags ( 6 cloud-based, 2 openstack) with the naming convention being `$version-$provider-$variant`. This is because there is no fundamental parameter differences required between versions/providers, and this behavior gives us confidence that we're testing the same configurations across different versions and/or platforms.
 
+The children of the "config" section tell where under config to look for the definitions.
```

Comment:
Great to see improvements in the documentation :)

Reply:
+1